teemip-framework/src/Model/_IPApplication.php

```diff
@@ -22,8 +22,8 @@ public function OnInsert() {
 		//
 		$oSearchDup = DBObjectSearch::FromOQL_AllData("SELECT IPDiscovery WHERE uuid LIKE :sUUID");
 		do {
-			$sId = strtoupper(md5(uniqid(rand(), true)));
-			$sFinalId = substr($sId, 0, 4).'_'.substr($sId, 4, 4).'_'.substr($sId, 8, 4).'_'.substr($sId, 12, 4);
+			$sId = strtoupper(bin2hex(random_bytes(8)));
+			$sFinalId = vsprintf("%s_%s_%s_%s", str_split($sId,4));
```

Comment:
I would prefer to see UUID's generated with a dash instead of underscore..

Reply:
OK to use dashes instead of underscores